torch/csrc/utils/python_arg_parser.h

```diff
@@ -162,8 +163,11 @@ struct PythonArgs {
   inline c10::optional<int64_t> toInt64Optional(int i);
   inline c10::optional<bool> toBoolOptional(int i);
   inline c10::optional<double> toDoubleOptional(int i);
-  inline const THPLayout& layout(int i);
-  inline const THPLayout& layoutWithDefault(int i, const THPLayout& default_layout);
+  // inline const at::Layout& layout(int i);
```

Comment:
nit: we can delete these, right? :)

Reply:
Thanks Ed! I also noticed it as well. I'm going to submit the change really quick.